files/assets/js/marked.custom.js

```diff
@@ -188,13 +188,14 @@ function safeMarkdown(input) {
 setTimeout(() => markdown('post-text','preview'), 200);
 
 function markdown(first, second) {
-	var input = document.getElementById(first).value;
-	
+	var input = document.getElementById(first);
 	var dest = document.getElementById(second);
-	for (var i = 0; i < dest.children.length; i++) {
-		dest.removeChild(dest.children[i]);
+	if(dest && input && input.value.trim() !== ''){
+		for (var i = 0; i < dest.children.length; i++) {
+			dest.removeChild(dest.children[i]);
+		}
+		dest.appendChild(safeMarkdown(input.value));
```

Comment:
![image](https://user-images.githubusercontent.com/57840444/172024212-2b7efe85-faad-449c-8cbc-0203460ffe57.png)

Something is causing the reformatting of the preview to go wrong. I'm assuming it's this stuff, but not 100%.

Reply:
According to TwoLargeSnakesMating on discord, this issue is found upstream in the rDrama codebase as well. I'm going to make an issue for it.

Reply:
The new issue is #105. I'm going to resolve this because I don't think it's related to this request.

Reply:
No problem, it doesn't seem like it's really a deal-breaker anyways for the site to function at all